crates/lgn-api-codegen/src/visitor.rs

```diff
@@ -508,11 +509,21 @@ impl Visitor {
         object_type: &OpenApiElement<'_, openapiv3::ObjectType>,
     ) -> Result<Type> {
         // New model is created on the fly for each object type.
-        if object_type.additional_properties.is_some() {
-            return Err(Error::Unsupported(
-                object_type.ref_().clone(),
-                "additional properties".to_string(),
-            ));
+        if let Some(ref additional_properties) = object_type.additional_properties {
+            let type_ = match additional_properties {
+                AdditionalProperties::Any(_) => {
+                    return Err(Error::Unsupported(
```

Comment:
We still don't support so called ["free-form objects"](https://swagger.io/docs/specification/data-models/dictionaries/#free-form).
Maybe `Box<dyn Any>` would be acceptable @ademange-legion  @ereOn ?

Reply:
Yeah, I guess that's as good as anything.

In Go, I used to have maps of (String, Interface{}) which is pretty much the equivalent.